config/prod/s3-synapse-sync.yaml

```diff
@@ -13,8 +13,9 @@ parameters:
   BucketVariables: >-
     {
       "htan-dcc-center-a":{"SynapseProjectId":"syn22343754"},
-      "htan-dcc-bu":{"SynapseProjectId":"syn22124336"},
-      "htan-dcc-ohsu":{"SynapseProjectId":"syn22093319"}
+      "htan-dcc-ohsu":{"SynapseProjectId":"syn22093319"},
```

Comment:
if you're intention is to delete `htan-dcc-bu` bucket then please delete the htan-dcc-bu.yaml file and include it in this PR.

Reply:
when I try to delete the file, it's treated as renaming to the new bucket. how can I avoid this?

Reply:
@zaro0508 should I just include the bucket deletion in a separate PR?

Reply:
yes, just do in a separate PR if you can't figure out the GIT thing.